src/components/Supplier/AppSupplierData/AddSupplierData.tsx

```diff
@@ -1,17 +1,213 @@
-import { Flex, Box } from '@chakra-ui/react';
-import AddSupplierDataName from './AddSupplierDataName';
-import AddSupplierDataAddress from './AddSupplierDataAddress';
+import {
+  Flex,
+  Box,
+  Container,
+  Grid,
+  Input,
+  Textarea,
+  NumberInput,
+  NumberInputField,
+  VStack,
+  StackDivider,
+} from '@chakra-ui/react';
+import { useForm } from 'react-hook-form';
+import AddSupplierButtons from 'components/Supplier/AddSupplierButtons';
+import styled from 'styled-components';
 
-const AddSupplierData = () => (
-  <Flex>
-    <Box w="80em" h="26em">
-      <AddSupplierDataName />
-    </Box>
-    <Box w="6em" h="26em" />
-    <Box w="100em" h="26em">
-      <AddSupplierDataAddress />
-    </Box>
-  </Flex>
-);
+const AddSupplierData = () => {
```

Comment:
@kacpergadula Try to make this component responsive. I saw that it is not right now.

Reply:
I corrected this